test/repl/proxysql_repl_tests/bin/debezium-init-config.bash

```
@@ -0,0 +1,22 @@
+# check if all required env vars are set
+if [ -z "${INFRA}" ]; then
+    echo "INFRA is empty, please check env vars passed to: " ${0}
+    exit 1
+elif [ -z "${USE_SSL}" ]; then
+    echo "USE_SSL is empty, please check env vars passed to: " ${0}
+    exit 1
+fi
+
+echo -n "Configuring connection: mysql1(source) => debezium(replication stream) ..."
+
+conn_mysql=`envsubst < conf/debezium/register-mysql1.json`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)